book/src/programs.md

```diff
@@ -48,18 +48,29 @@ read its data and credit the account.
 
 In the same way that a Linux user uses a path to look up a file, a Solana
 client uses public keys to look up accounts. To create an account, the client
-generates a *keypair* and registers its public key using the CreateAccount
-instruction. Once registered, transactions reference account keys to grant
-programs access to accounts. The runtime grants programs read access by
-default. To grant write access, the client must either assign the account to a
-program or sign the transaction using the keypair's *secret key*. Since only
-the holder of the secret key can produce valid signatures matching the
-account's public key, the runtime recognizes the signature as authorization to
-modify account data or debit the account.
+generates a *keypair* and registers its public key using the `CreateAccount`
+instruction. The account created by `CreateAccount` is called a *user account*
```

Comment:
As far as I can tell, the only place we call these user accounts is in the `create_user_account()` method. Other places we call them system accounts.
Based on how we use other modifiers to "account", I think system account is more correct... but maybe you are  trying to distinguish something different (than the owning program) with "user"?